pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -41,12 +41,13 @@ const (
 )
 
 type StorageInitializerConfig struct {
-	Image                 string `json:"image"`
-	CpuRequest            string `json:"cpuRequest"`
-	CpuLimit              string `json:"cpuLimit"`
-	MemoryRequest         string `json:"memoryRequest"`
-	MemoryLimit           string `json:"memoryLimit"`
-	StorageSpecSecretName string `json:"storageSpecSecretName"`
+	Image                      string `json:"image"`
+	CpuRequest                 string `json:"cpuRequest"`
+	CpuLimit                   string `json:"cpuLimit"`
+	MemoryRequest              string `json:"memoryRequest"`
+	MemoryLimit                string `json:"memoryLimit"`
+	StorageSpecSecretName      string `json:"storageSpecSecretName"`
+	EnableDirectPvcVolumeMount string `json:"enableDirectPvcVolumeMount"`
```

Comment:
@lizzzcai use the boolean type like here ?https://github.com/kserve/kserve/blob/e3a10812671f816565120fa19f90ab667a30252b/pkg/apis/serving/v1beta1/configmap.go#L77

Reply:
https://github.com/kserve/kserve/pull/2738#discussion_r1137238608
as discussed here I make it as a string to follow other example, but from the example you provided here there are different format exists in KServe, in this case, I will make it `bool` which make more sense.

Reply:
Thanks! We will fix the other places to be consistent.